Makefile

```
@@ -766,3 +766,11 @@ $(BINARY_DEPS_DIR)/linux-%:
 ifneq ($(FETCH_BINARIES_TARGETS),)
 .SECONDARY: $(call FULL_FETCH_BINARIES_TARGETS, $(FETCH_BINARIES_TARGETS))
 endif
+
+E2E_BINARY?=./bin/e2e.test
```

nit: we can drop the `./`

```suggestion
E2E_BINARY?=bin/e2e.test
```